recipes/espnet/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "espnet" %}
+{% set version = "202205" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/espnet-{{ version }}.tar.gz
+  sha256: c7bfc2e76a25bf67b82de7165e201861c8ca061e413715583bf84cb963576197
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - numpy <=1.21.3
+    - pip
+    - pytest-runner
+    - python >=3.6
+  run:
+    - ci-sdr
+    - configargparse >=1.2.1
+    - ctc-segmentation >=1.6.6
+    - espnet-tts-frontend
+    - fast-bss-eval ==0.1.3
+    - filelock
+    - h5py >=2.10.0
+    - humanfriendly
+    - jamo >=0.4.1
+    - kaldiio >=2.17.0
+    - librosa >=0.8.0
+    - nltk >=3.4.5
+    - pypinyin <=0.44.0
+    - pysoundfile >=0.10.2
+    - python >=3.6
```

```suggestion
    - python >=3.7
  run:
    - ci-sdr
    - configargparse >=1.2.1
    - ctc-segmentation >=1.6.6
    - espnet-tts-frontend
    - fast-bss-eval ==0.1.3
    - filelock
    - h5py >=2.10.0
    - humanfriendly
    - jamo >=0.4.1
    - kaldiio >=2.17.0
    - librosa >=0.8.0
    - nltk >=3.4.5
    - pypinyin <=0.44.0
    - pysoundfile >=0.10.2
    - python >=3.7
```
https://github.com/espnet/espnet/blob/5fa6dcc4e649dc66397c629d0030d09ecef36b80/setup.py#L136